### PR TITLE
[BUGFIX] Fixed bug in media list: disabled sorting of thumbnails

### PR DESCRIPTION
**Pull request by turbo-ele**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | none |
| Related issues/PRs | none |
| License | MIT |
| Documentation PR | none |
#### What's in this PR?

Disabling sort of thumbnails in media lists.
#### Why?

Thumbnails cannot be sorted. An exception is thrown.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js [hunk 1]
```diff
@@ -102,12 +102,12 @@
...
              * loads an url and matches it against user settings
              * @param key Defines which setting to compare with
-             * @param excludeAttributes Defines which Attributes should NOT be taken from user-settings and from fields API instead
+             * @param includedAttributes Defines which Attributes should  be taken from user-settings and from fields API instead
              * @param url Where
              * @param callback
              * @param filter callback to filter result
              */
-            app.sandbox.sulu.loadUrlAndMergeWithSetting = function(key, excludeAttributes, url, callback, filter) {
+            app.sandbox.sulu.loadUrlAndMergeWithSetting = function(key, includedAttributes, url, callback, filter) {
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js]: @turbo-ele i have no idea what are you doing here (:

File: src/Sulu/Bundle/MediaBundle/Resources/config/list-builder/Media.xml [hunk 2]
```diff
@@ -55,7 +55,8 @@
             <orm:entity-name>SuluMediaBundle:Media</orm:entity-name>
         </identity-property>
 
-        <property name="thumbnails" list:translation="media.media.thumbnails" display="always" list:type="thumbnails"/>
+        <property name="thumbnails" list:translation="media.media.thumbnails" display="always" list:sortable="false"
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/config/list-builder/Media.xml]: this will be fixed by https://github.com/sulu/sulu/pull/2412

**Comment by danrot** [comment]
@turbo-ele Do you still need that change? The bug you have described here does not appear anymore.

**Comment by turbo-ele** [comment]
the original issue was that thumbnails were sortable, which was fixed in #2412 as @wachterjohannes pointed out.

Second thing which came into my attention was, that the javascript functionality for merging user settings with results of fields from api is using a blacklisting approach. Which leaded to an issue, that a field (which i can't remember the name of) was merged with settings. I think the only field which actually needs to be synced with user-settings is the `disabled` property, everything else should be taken from api.

**Comment by chirimoya** [comment]
ping @wachterjohannes @danrot

**Comment by wachterjohannes** [comment]
@turbo-ele can you finish this PR with your second thought (: and also update the fescription of this!

**Comment by danrot** [comment]
ping @turbo-ele

**Pull request closed without merging**